InvenTree/company/templates/company/supplier_part.html

```diff
@@ -268,6 +268,11 @@ <h4>{% trans "Pricing Information" %}</h4>
                 return html;
             }
         },
+        {
+            field: 'updated',
+            title: '{% trans "Last updated" %}',
+            sortable: true,
```

Comment:
Please ensure that this date value is rendered through the `renderDate` function, according to user preference.

e.g.

```
  formatter: function(value) {
      return renderDate(value);
  }
```